### PR TITLE
HELM : Support priorityClassName

### PR DESCRIPTION
**Pull request by krlendon**
### What problem does this PR solve?
Currently you cannot pass priorityClassName to set a pod priority on deployed pods/daemonsets. This allows you to specify a custom priorityClassName if desired.

### What is changed and how does it work?
added the field priorityClassName to the template spec for deployments/daemonset to allow custom values.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [X] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
please run `hack/update_install_script.sh` to update the `install.sh`

**Comment by krlendon** [comment]
Ran that script and pushed. Now getting a boilerplate error.

File: hack/install.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (WangXiangUSTC) [inline hack/install.sh:1]: this file can be deleted

**Comment by zhouqiang-cl** [comment]
@krlendon thank you for your contribution

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by Gallardot — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**